### DISCUSSION TITLE
Does this package support RAG?

### DISCUSSION THREAD

**Discussion by Jamie123234** [BrutalCoding/aub.ai#23] [Ideas]
I currently use the Gemini API(online) to process my invoices. The API model has constantly become overloaded and is not stable. Using a local LLM would solve all problems, however it does it support file processing like retrieval augmented generations?

**Comment by BrutalCoding** [comment]
Heya @Jamie123234, no RAG isn't supported out of the box. 

Just a quick heads up - I’ll keep working on this package, but I don’t have a set timeline yet.

I took a break because the mental strain I put on myself, along with some personal stuff, got a bit overwhelming.
